tests/storage/conftest.py

```diff
@@ -425,6 +426,16 @@ def cirros_vm_name(request):
     return request.param["vm_name"]
 
 
+@pytest.fixture(scope="module")
+def rhel_data_source_scope_module(golden_images_namespace):
```

Comment:
```suggestion
def rhel10_data_source_scope_module(golden_images_namespace):
```

Note there is this PR which also adds the same fixture, so which ever gets merged later will need to merge them  - https://github.com/RedHatQE/openshift-virtualization-tests/pull/2350